.vscode/tasks.json

```diff
@@ -255,16 +138,16 @@
 				"close": false
 			},
 			"dependsOn": [
-				"K3D - Deploy runtime",
-				"K3D - Build VehicleApp"
+				"K3D Runtime - Deploy runtime",
+				"K3D Runtime - Build VehicleApp"
```

Comment:
Should that be removed?
```suggestion
```

Reply:
Deploy-vehicleapp is already checking internally if the app is build, so we don't need this dependency